packages/core/src/index.tsx

```diff
@@ -126,12 +126,12 @@ export function App({ config, initialResponse }: AppProps): ReactElement {
       overlayCloseListener.current = onClose;
     }
 
+    setOverlayCloseRequested(false);
```

Comment:
Just moved this here in case the above code sets `overlayCloseRequested` while the overlay was closed.